### PR TITLE
[Content Layout] Putting together content layout

### PR DESCRIPTION
**Pull request by DLotero**
## Summary

-Putting together whole content layout with all the technologies as per design

## Screenshots

image

## Trello Card
https://trello.com/c/12B49Kkd/23-content-layout

### CODE REVIEW

File: maquetacion/scss/techs.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+.techs {
+  display: flex;
+  flex-flow: row wrap;
+  margin-left: 80px;
+  width: 90%;
```
Comment (FrankIglesias) [inline maquetacion/scss/techs.scss]: use px instead

File: maquetacion/scss/techs.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+.tech-card {
+  align-items: center;
+  background-color: $earls-green;
+  border-radius: 5px;
+  display: flex;
+  flex-flow: column nowrap;
+  flex-basis: 20%;
```
Comment (FrankIglesias) [inline maquetacion/scss/techs.scss]: why this?

File: maquetacion/index.html [hunk 3]
```diff
@@ -1,17 +1,99 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML & CSS Training</title>
-    <meta name="description" content="training">
-    <meta name="author" content="Wolox">
-    <link rel="stylesheet" type="text/css" href="application.css">
-  </head>
+
```
Comment (FrankIglesias) [inline maquetacion/index.html]: delete line

File: maquetacion/index.html [hunk 4]
```diff
@@ -1,17 +1,99 @@
...
+        </div>
+      </div>
+    </section>
+    <section class="techs-section">
+      <h1 class="titles">
+        Back-End Tools
+      </h1>
+      <div class="techs">
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/rails_icon.png" alt='Ruby Logo'>
```
Comment (FrankIglesias) [inline maquetacion/index.html]: use double comma please

File: maquetacion/index.html [hunk 5]
```diff
@@ -1,17 +1,99 @@
...
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/express_icon.svg" alt='Node Logo'>
+          <h1 class="tech-card-name">NodeJS</h1>
+          <h2 class="tech-card-date">Release: 2013</h2>
+          <p class="tech-card-info">
+            React is a declarative, component-based Javascript library for building user interfaces
+          </p>
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/aspnet_icon.png" alt='ASP.NET Logo'>
```
Comment (FrankIglesias) [inline maquetacion/index.html]: same

File: maquetacion/index.html [hunk 6]
```diff
@@ -1,17 +1,99 @@
...
+          <h1 class="tech-card-name">ASP.NET Core</h1>
+          <h2 class="tech-card-date">Release: 2013</h2>
+          <p class="tech-card-info">
+            React is a declarative, component-based Javascript library for building user interfaces
+          </p>
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/django_icon.svg" alt='Django Logo'>
```
Comment (FrankIglesias) [inline maquetacion/index.html]: same

File: maquetacion/index.html [hunk 7]
```diff
@@ -1,17 +1,99 @@
...
+          <h2 class="tech-card-date">Release: 2013</h2>
+          <p class="tech-card-info">
+            React is a declarative, component-based Javascript library for building user interfaces
+          </p>
+        </div>
       </div>
     </section>
   </body>
-</html>
+</html>
```
Comment (FrankIglesias) [inline maquetacion/index.html]: add tailing line

File: maquetacion/scss/titles.scss [hunk 8]
```diff
@@ -1,21 +1,18 @@
...
-
-.titles-text {
-  position: absolute;
-  width: 100%;
-  left: 11%;
-  top: 41%;
-  font-size: 18px;
-  z-index: 2;
-}
+
```
Comment (FrankIglesias) [inline maquetacion/scss/titles.scss]: remove space

**Review by FrankIglesias — CHANGES_REQUESTED** [review]
use double commas in alts

File: maquetacion/scss/techs.scss (hunk 2 above)
Comment (DLotero) [reply maquetacion/scss/techs.scss]: to make sure only 4 cards show at the same line at any given time :D

File: maquetacion/index.html [hunk 9]
```diff
@@ -1,16 +1,92 @@
 <!doctype html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <title>HTML & CSS Training</title>
-    <meta name="description" content="training">
-    <meta name="author" content="Wolox">
-    <link rel="stylesheet" type="text/css" href="application.css">
-  </head>
+<head>
```
Comment (FrankIglesias) [inline maquetacion/index.html]: indent head

File: maquetacion/index.html [hunk 10]
```diff
@@ -1,16 +1,92 @@
...
-    <section class='titles'>
-      <div class='titles-content'>
-        <div class='titles-text'>Front-End Tools</div>
+    <section class="techs-section">
+      <h1 class="titles">
+        Front-End Tools
+      </h1>
+      <div class="techs">
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/react_icon.svg" alt="React Logo">
```
Comment (FrankIglesias) [inline maquetacion/index.html]: close the img tag at the end like this
``

File: maquetacion/index.html [hunk 11]
```diff
@@ -1,16 +1,92 @@
...
+          <h1 class="tech-card-name">ReactJS</h1>
+          <h2 class="tech-card-date">Release: 2013</h2>
+          <p class="tech-card-info">
+            React is a declarative, component-based Javascript library for building user interfaces
+          </p>
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/angular_icon.svg" alt="Angular Logo">
```
Comment (FrankIglesias) [inline maquetacion/index.html]: here

File: maquetacion/index.html [hunk 12]
```diff
@@ -1,16 +1,92 @@
...
+          <h1 class="tech-card-name">AngularJS</h1>
+          <h2 class="tech-card-date">Release: 2010</h2>
+          <p class="tech-card-info">
+              AngularJS (commonly reffered to as AngularJS 1.X) is a JavaScript framework for building SPAs with an MVC architecture.          </p>
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/vue_icon.svg" alt="Vue Logo">
```
Comment (FrankIglesias) [inline maquetacion/index.html]: here

File: maquetacion/index.html [hunk 13]
```diff
@@ -1,16 +1,92 @@
...
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/elm_icon.svg" alt="Elm Logo">
+          <h1 class="tech-card-name">Elm</h1>
+          <h2 class="tech-card-date">Release: 2016</h2>
+          <p class="tech-card-info">
+              Elm is a domain-specific programming language for declaratively creating web browser-based graphical user interfaces.          </p>
+        </div>
+        <div class="tech-card">
+          <img class="tech-card-logo" src="./assets/tech-logos/polymer_icon.svg" alt="Polymer Logo">
```
Comment (FrankIglesias) [inline maquetacion/index.html]: same

File: maquetacion/scss/styles.scss [hunk 14]
```diff
@@ -3,3 +3,7 @@
 body {
   font-family: 'Roboto', sans-serif;
 }
+
+.techs-section {
```
Comment (FrankIglesias) [inline maquetacion/scss/styles.scss]: move this class to techs file

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**